geopandas/geoseries.py

```diff
@@ -173,8 +222,15 @@ def copy(self, order='C'):
         copy : GeoSeries
         """
         # FIXME: this will likely be unnecessary in pandas >= 0.13
-        return GeoSeries(self.values.copy(order), index=self.index,
-                      name=self.name).__finalize__(self)
+        return GeoSeries(self._data,
+                         index=self.index, crs=self.crs,
+                         name=self.name)
```

Comment:
A copy of the data, so in our case it means taking a copy of all the actual c objects? a VectorizeGeometry.copy ? method